src/battle/battle.c

```diff
@@ -381,6 +385,30 @@ static void handleKeyboard(void)
 	}
 }
 
+static void handleController(void)
+{
+	if ((app.controllerAxis[0] != 0) || (app.controllerAxis[0] != 0) || (app.controllerAxis[0] != 0) || (app.controllerAxis[0] != 0))
+	{
+		if (app.controllerX == CONTROLLER_NOINPUT)
+		{
+			app.controllerX = app.mouse.x;
+			app.controllerY = app.mouse.y;
+		}
+		app.controllerX += app.controllerAxis[0] / 4;
+		if (app.controllerX < 0)
```

Comment:
Please add braces to all the single line if statements (including the else ifs).

Reply:
Done.